numpy/core/fromnumeric.py

```diff
@@ -721,7 +720,8 @@ def partition(a, kth, axis=-1, kind='introselect', order=None):
     ================= ======= ============= ============ =======
        kind            speed   worst case    work space  stable
     ================= ======= ============= ============ =======
-    'introselect'        1        O(n)           0         no
+    'introselect'        2        O(n)           0         no
+    'floydselect'        1........O(n^2)         0         no
```

Comment:
This looks like invalid rST:
```suggestion
    'floydselect'        1        O(n^2)         0         no
```